client/state/analytics/middleware.js

```diff
@@ -0,0 +1,40 @@
+import analytics from 'analytics';
+import invoke from 'lodash/invoke';
+import reduxAnalytics from 'redux-analytics';
```

Comment:
Well, buy me slippers and call me Dorothy!

I would just write it from scratch, but theres a module for it in `package.json`!
Yeah, we should really get this PR some tracktion